mrp_workcenter_workorder_link/mrp_view.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+<!-- mrp.workcenter -->
+
+<record id="mrp_workcenter_tree_view" model="ir.ui.view">
+    <field name="model">mrp.workcenter</field>
+    <field name="inherit_id" ref="mrp.mrp_workcenter_tree_view"/>
+    <field name="arch" type="xml">
+        <field name="name" position="before">
+            <button name="button_workcenter_line"
+                    string="Click to see workcenter's operations"
+                    icon="gtk-execute"
+                    type="object"/>
```

Comment:
I'd rather have a normal big red button at the top of the form, like the standard "Delivery Order" or "Invoice" buttons on Sale Orders, Purchase Orders etc.